admin/installer/setup_client.sh

```diff
@@ -13,7 +13,7 @@ apt-get update
 sudo apt-get install -y postgresql-client
 
 # Get docker-compose.
-curl -L https://github.com/docker/compose/releases/download/1.5.2/docker-compose-`uname -s`-`uname -m` > /usr/local/bin/docker-compose
+curl -L https://github.com/docker/compose/releases/download/1.7.1/docker-compose-`uname -s`-`uname -m` > /usr/local/bin/docker-compose
```

Comment:
We should probably pre-install this in the image to reduce the risk of failures caused by flaky github services or network connections.